inst/templates/ad_adrs.R

```diff
@@ -85,11 +85,45 @@ adrs <- adrs %>%
       mode = "last"
     ),
     filter = !is.na(AVAL) & ADT >= RANDDT
+  ) %>%
+  derive_var_relative_flag(
+    by_vars = exprs(STUDYID, USUBJID),
+    order = exprs(ADT, RSSEQ),
+    new_var = ANL02FL,
+    condition = AVALC == "PD",
+    mode = "first",
+    selection = "before",
+    inclusive = TRUE
   )
 
-# ---- Parameter derivations ----
+# Create dataset with overall responses to be used for deriving parameters
+ovr <- filter(adrs, PARAMCD == "OVR" & ANL01FL == "Y" & ANL02FL == "Y")
+
+# Parameter derivations ----
+
+## Define events ----
```

Comment:
People might get confused why we include some of these here. Would it make sense to add an extra comment to explain these are just some examples given where a user may create their own versions of these source objects compared to those provided in R/response_sources.R, in this case as needed to keep all ADSL vars

Reply:
I have added a comment.